packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -88,6 +89,10 @@ abstract contract MajorityVotingBase is
     /// @param minDuration The minimal duration of the vote in seconds.
     error VotingPeriodInvalid(uint64 current, uint64 start, uint64 end, uint64 minDuration);
 
+    /// @notice Thrown when a sender is not allowed to create a vote.
+    /// @param sender The sender address.
+    error VoteCreationForbidden(address sender);
```

Comment:
this error is used in `createProposal` in tokenvoting so I assume that it should be named `ProposalCreationForbidden` ?

Reply:
Fixed in bada99eca8f86d81fb8a0ebe3c1d963e560547f9